common/ellipses.js

```diff
@@ -293,13 +301,13 @@
 
```

Comment:
[readmore](https://github.com/informatics-isi-edu/chaise/blob/rec-errors/common/ellipses.js#L288) should be wrapped.